go/go.sum

```diff
@@ -780,6 +783,7 @@ github.com/oasisprotocol/deoxysii v0.0.0-20200527154044-851aec403956 h1:etZXZf8f
 github.com/oasisprotocol/deoxysii v0.0.0-20200527154044-851aec403956/go.mod h1:cE5EgXTIhq5oAVdZ7LZd1FjTRLALPEzv93CWzBtDkyI=
 github.com/oasisprotocol/ed25519 v0.0.0-20200528083105-55566edd6df0 h1:qmiMZ6ZhkeQZkV/Huajj+QBAu1jX0HTGsOwi+eXTGY8=
 github.com/oasisprotocol/ed25519 v0.0.0-20200528083105-55566edd6df0/go.mod h1:IZbb50w3AB72BVobEF6qG93NNSrTw/V2QlboxqSu3Xw=
+github.com/oasisprotocol/oasis-core/go v0.0.0-20200623153002-9e61aea5195b/go.mod h1:/8dnlXMWVE88E6q4Dq/JoQyQhBfJzHtnebCmmPu2SLI=
```

Comment:
I don't like this circular dependency as I'm not sure how this will interact with modules, but let's accept this for now and we'll fix it later.